### PR TITLE
Adding more 'work' choices

### LINKED ISSUE #322: Add "retired" category
Add a new category named "retired" for the showcase. It should be below `C4DT Factory Involved`, and perhaps be named `Retired from C4DT Factory`.

### PR DESCRIPTION
**Pull request by ineiti**
Instead of only having 'C4DT Factory involved', add more sub-categories:
- market
- retired
- archived

Care has been taken that the display makes sense: the 'retired' work choice shows both retired and archived artefacts.

Closes #322

### CODE REVIEW

File: views/projects.tpl [hunk 1]
```diff
@@ -75,7 +75,8 @@
 
             // Set focus on the search input
             $('#search').focus();
-            table.draw();
+            // table.draw();
```
Comment (PascalinDe) [inline views/projects.tpl:78]: please remove commented code

File: views/projects.tpl [hunk 2]
```diff
@@ -94,7 +95,7 @@
 
             ["work", "categories", "applications", "lab"].forEach(id => {
                 let select = $(`#${id}`)[0];
-		  select.selectedIndex = 0;
+		        select.selectedIndex = 0;
```
Comment (PascalinDe) [inline views/projects.tpl:98]: please undo automated indent

Comment (ineiti) [reply views/projects.tpl]: tab / space mixup, I'm surprised that `prettier` didn't catch this.

File: views/projects.tpl [hunk 3]
```diff
@@ -232,7 +233,10 @@ applications.update({ "Other": "Other" })
                         style="width: 13em;"
                         onchange="search_apply();">
                     <option selected value="">All projects</option>
-                    <option value="project_incubated">C4DT Factory involved</option>
+                    <option value="project_incubated">C4DT Factory currently involved</option>
+                    <option value="project_incubated_market">C4DT Factory actively supported</option>
+                    <option value="project_retired">Retired C4DT Factory projects</option>
+                    <option value="project_retired_archived">Archived C4DT Factory projects</option>
                     <option value="project_active">Updated in last 6 months</option>
```
[the remark this answers is not in the document]
Comment (PascalinDe) [reply views/projects.tpl:239]: ah, I see, no, nothing comes to mind, let's leave it like this then

**Review by PascalinDe — APPROVED** [review]

**Pull request merged**